docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -257,3 +261,4 @@ for this).
 0. Make sure that you can run your tests against the Simulator.
 0. Make sure UI Automation is enabled on your device. Settings -> Developer ->
    Enable UI Automation
+0. If you are using appium desktop app, chekcout the details logs to find your appium folder.
```

Comment:
This description looks not very clear. It would be better to describe where these logs can be found in the app (provide exact menu item names).